scripts/go-release-automation.sh

```diff
@@ -64,17 +66,11 @@ run_release_script() {
     echo "Removed all shim.go files"
   fi
 
-  echo "Running goimports..."
-  goimports -w .
+  run_go_tools
 
-  echo "Running go get -u..."
-  go get -u
-
-  echo "Running go mod tidy..."
-  go mod tidy
-
-  echo "Running go build to check for errors..."
-  go build --gcflags="-e" ./...
+  # Replacement directives are removed to get package from go pkg instead of local copy
```

Comment:
```suggestion
  # Replacement directives are removed to get package from go pkg instead of local copy
  # We need to copy `go.mod` otherwise without it this script will not work in the first release because all the go tools used in this script works only when there is a `go.mod` in the directory.
```